src/views/Deposit/index.tsx

```diff
@@ -4,37 +4,28 @@ import React from 'react';
 import { ReactComponent as SelectorSvg } from 'assets/icons/selector.svg';
 
 // Components
-import {
-    EnterAmount,
-    InfoCard,
-    StyledButtonLarge,
-    StyledSelectorContainer,
-} from './components';
+import { EnterAmount, InfoCard, StyledButtonLarge, StyledSelectorContainer } from './components';
 import { Card, InnerCardContainer } from 'components/Card';
 import { Flex } from 'components/Containers';
 import Spacer from 'components/Spacer';
 
 const Deposit: React.FC = () => {
     // State
```

Comment:
Let's also remove this comment—nothing useful here

Reply:
Fixed in 08968fd7fafce8b01422684b616129de36097a05